index.html

```diff
@@ -17,148 +17,255 @@
             <a class="logo-container" href="">
                 <img src="images/image_geometry_1logo.png" alt="logo">
             </a>
-
             <nav>
+                <a href="#Hello">Hello</a>
+                <a href="#projects">Portfolio</a>
+                <a href="#about">About</a>
+                <a href="#contact">Contact</a>
+
                 <a class="a-logo" href=""><i class="fas fa-bars"></i></a>
             </nav>
         </header>
         <section class="headline">
             <div class="name">
                 <h1>Gabriel <br> Nwachukwu</h1>
             </div>
+            <div class="geometry_desktop"></div>
             <div class="main">
                 <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
 
             </div>
             <button class="collaboration" type="button">
                 start collaboration
             </button>
+            <div class="image-dots"></div>
             <div class="geometry"></div>
         </section>
-        <section class="work-section-mobile-grid">
+        <section class="mobile-grid-container">
             <div class="project-title">
-                <p>Projects</p>
+                <p>project</p>
             </div>
-            <div class="card-container-grid">
-                <!-- this is a background image tag -->
-                <div class="bg-screen bg-img"></div>
-                <div class="project-name card-details">
-                    <h3>Project name goes here..</h3>
-                    <ul class="languages">
-                        <li class="html language">Html/css</li>
-                        <li class="rails language">Ruby/rails</li>
-                        <li class="js language">Javascript</li>
-                    </ul>
-                    <a class="see-this-project" href="">
+            <!-- this is a background image tag -->
+
+            <div class="project-container">
+                <div class="project-name">
+                    <h3>Welcome microverse</h3>
+                    <div class="languages">
+                        <div class="html">Html/css</div>
+                        <div class="rails">Ruby/Rails</div>
+                        <div class="js">Javascript</div>
+                    </div>
+                    <a class="see-this-project" href="/">
```

Comment:
- [ ]  For all your button you should consider applying the styles when you hover or click a button (it depends on the background color) What I recommend you is only adding the second and third columns in your app using the next CSS selectors:
```
// To add styles when you hover buttons
.button-class:hover {
  // Change color, border color, background-color.
}

// To add styles when you click buttons
.button-class:active { 
  // Change color, border color, background-color.
}
```

Button styles from Figma design you selected:

![image](https://user-images.githubusercontent.com/60164239/131208190-5a1b8f04-f060-444d-b911-3507c4bccaaf.png)